### PR TITLE
Amam/enable translation

### PR DESCRIPTION
**Pull request by mustofa-binary**
# Description

Enable translation and add Content security policy

Changes:

-   enable translation

## Type of change

-   [ ] Bug fix
-   [x] New feature
-   [x] Breaking change
-   [ ] Dependency update
-   [ ] This change requires a documentation update

### CODE REVIEW

**Comment by oskar-binary** [comment]
can you show the language dropdown for non-production also? + send the selected language to the socket connection in (socket_base)

**Comment by mustofa-binary** [comment]
> can you show the language dropdown for non-production also? + send the selected language to the socket connection in (socket_base)

we dont have design for language dropdown yet, sure will send to socket base

File: src/common/utility.js [hunk 1]
```diff
@@ -38,6 +38,10 @@ const getPropertyValue = (obj, k) => {
     return obj ? cloneObject(obj[keys[0]]) : undefined
 }
 const getLocationHash = () => (location.hash ? location.hash.substring(1) : '')
+
+const getLanguage = () =>
+    typeof localStorage !== 'undefined' ? localStorage.getItem('i18n') : null
```
Comment (oskar-binary) [inline src/common/utility.js]: should return `en` as default I think if there is no `i18n` in localStorage

File: src/pages/ach.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react'
```
Comment (oskar-binary) [inline src/pages/ach.js]: I don't think `ach` should be a separate page 🤔 I think it's enough to enable in-context translation if we change the current language but keep the base same as english `deriv.com` if that is possible

File: i18n-config.js [hunk 3]
```diff
@@ -6,48 +6,48 @@ module.exports = {
...
+        path: 'it',
+        display_name: 'Italiano',
+    },
+    pl: {
+        path: 'pl',
+        display_name: 'Polish',
+    },
+    de: {
+        path: 'de',
+        display_name: 'Português',
```
Comment (oskar-binary) [inline i18n-config.js]: `de` should be German

**Comment by oskar-binary** [comment]
deployment is failing

File: i18n-config.js [hunk 4]
```diff
@@ -6,48 +6,56 @@ module.exports = {
         display_name: 'English',
     },
     // TODO: uncomment this when translations are ready
```
Comment (oskar-binary) [inline i18n-config.js]: we can remove this comment now also

File: src/components/localization/config.js [hunk 5]
```diff
@@ -8,19 +8,20 @@ i18n.use(initReactI18next).init({
     // TODO: Optimize translation file loading with GraphQL
     // TODO: Enable this when translations are ready
```
Comment (oskar-binary) [inline src/components/localization/config.js]: and these 2

**Pull request merged**